DemoApp/StreamChat/Components/DemoChatMessageActionsVC.swift

```diff
@@ -3,19 +3,48 @@
 //
 
 import Foundation
+import StreamChat
 import StreamChatUI
 import UIKit
 
 final class DemoChatMessageActionsVC: ChatMessageActionsVC {
     // For the propose of the demo app, we add an extra hard delete message to test it.
     override var messageActions: [ChatMessageActionItem] {
         var actions = super.messageActions
-        if message?.isSentByCurrentUser == true && AppConfig.shared.demoAppConfig.isHardDeleteEnabled {
-            actions.append(hardDeleteActionItem())
+        if message?.isSentByCurrentUser == true {
```

Comment:
Why don't we allow pinning someone's messages? 🙂

Reply:
We need to check for channel permission `PinMessage` but we don't have channel permissions implemented yet. Baby steps :)